examples/pod_runtime.json

```diff
@@ -10,7 +10,7 @@
             },
             "app": {
                 "exec": [
-                    "/bin/reduce-worker",
+                    "/usr/bin/reduce-worker",
```

Comment:
to match https://github.com/appc/spec/pull/496/files#diff-b13ba2410d4461ffc5242e15375b6cf5R13